nncf/quantization/algorithms/accuracy_aware/utils.py

```diff
@@ -0,0 +1,78 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import Callable
+from typing import Tuple
+from typing import List
+
+from nncf.common.graph import NNCFGraph
+from nncf.common.graph import NNCFNode
+
+
+def find_fq_nodes_to_cut(graph: NNCFGraph,
+                         fq_node: NNCFNode,
+                         is_fake_quantize: Callable[[NNCFNode], bool],
```

Comment:
What is the benefit of using a function instead of a metatype list?

Reply:
It is more flexible. For example, I can check a node's attribute to make a decision about a node. But I agree, that in this case, I could use lists of meta types.